Modelica/Blocks/Logical.mo

```diff
@@ -945,14 +945,16 @@ The output <code>Q</code> is set by the input <code>S</code>, is reset by the in
       Documentation(
         info="<html>
 <p>
-This block is used by connecting a&nbsp;(possibly) <strong>time varying</strong>
-input condition for the Boolean input variable&nbsp;<strong>u</strong>.
-If this input variable is <strong>false</strong>, the simulation will
+This block asserts that the Boolean input&nbsp;<strong>u</strong> is <strong>true</strong>.
+If&nbsp;<strong>u</strong> is <strong>false</strong>, the simulation will
 either abort with an error (if <strong>assertionLevel</strong> is set to
 <strong>error</strong>), or issue a&nbsp;warning and continue (if
 <strong>assertionLevel</strong> is set to <strong>warning</strong>).
-The <strong>message</strong> will be included in the message text of
-the error or warning.
+The <strong>message</strong> will be reported in either case.
+Typically, this block is used by connecting a&nbsp;(possibly) <strong>time
```

Comment:
How about a paragraph break before normative description and typical usage?

Reply:
I can even imagine to change the last two sentences, so that "For further information, ..." goes to the first paragraph and "Typically, this block..." is the second paragraph.
@henrikt-ma Would you make a commit?